homeassistant/components/androidtv/media_player.py

```diff
@@ -475,6 +477,35 @@ def unique_id(self):
         """Return the device unique id."""
         return self._unique_id
 
+    async def async_get_media_image(self):
+        """Fetch current playing image."""
+        media_data = await self.hass.async_add_executor_job(self.get_raw_media_data)
+        if media_data:
+            return media_data, "image/png"
+        return None, None
+
+    @adb_decorator()
+    def get_raw_media_data(self):
+        """Raw base64 image data."""
+        if self.state not in [STATE_OFF, None]:
```

Comment:
let's make this a guard clause.

```suggestion
        if self.state in [STATE_OFF, None]:
            return None
```

But probably even better to not even call this to begin with, so put this guard in `async_get_media_image` so we don't have to schedule a job for the executor.